### PR TITLE
update Observability diagrams

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- update diagrams related to observability docs
- ...
- ...

**Related issue(s)**

part of https://github.com/kyma-project/kyma/issues/11228
guidelines as in https://github.com/kyma-project/community/issues/542

### CODE REVIEW

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 title: Tracing Architecture
 ---
 
-
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:5]: Collect traces flow: 
Merge point 2 and 3, summarizing that it eithers goes through port-forwarding directly to Jaeger or via ingress passing the oauth2-proxy, which assures authenticated users via configured OIDC provider

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md [hunk 2]
```diff
@@ -4,15 +4,17 @@ title: Monitoring Architecture
 
 ## End-to-end monitoring flow
 
-The complete monitoring flow in Kyma comes down to these components and steps:
+The monitoring flow in Kyma comes down to the following components and steps:
 
 ![End-to-end monitoring flow](./assets/obsv-monitoring-flow.svg)
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:9]: The text on the arrow from operator to alertmanager is missing. It should be the same text as on the other arrow

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md [hunk 3]
```diff
@@ -4,15 +4,17 @@ title: Monitoring Architecture
...
 
-1. Upon Kyma installation on a cluster, Prometheus Operator creates a Prometheus instance with the default configuration.
-2. The Prometheus server periodically polls all metrics exposed on `/metrics` endpoints of ports specified in ServiceMonitor CRDs. Prometheus stores these metrics in a time-series database.
-3. If Prometheus detects any metric values matching the logic of alerting rules, it triggers the alerts and passes them to Alertmanager.
-4. If you manually configure a notification channel, you can instantly receive detailed information on metric alerts detected by Prometheus.
-5. You can visualize metrics and track their historical data on Grafana dashboards.
+1. Upon Kyma installation on a cluster, **Prometheus Operator** creates a **Prometheus** instance with the default configuration.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:11]:
```suggestion
1. Upon Kyma installation on a cluster, **Prometheus Operator** creates a **Prometheus** and a **Alertmanager** instance with the default configuration.
```

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md [hunk 4]
```diff
@@ -21,9 +23,9 @@ The diagram presents monitoring components and the way they interact with one an
 ![Monitoring components](./assets/obsv-monitoring-architecture.svg)
 
 
-1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a Prometheus instance, manages its deployment, and provides configuration for it. It also deploys Alertmanager and operates ServiceMonitor custom resources that specify monitoring definitions for groups of services.
+1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a **Prometheus** instance, manages its deployment, and provides configuration for it. It also deploys **Alertmanager** and operates **ServiceMonitor** custom resources that specify monitoring definitions for groups of services.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:26]: One would operate an active component only. Here it is a resource, so managing is the better term
```suggestion
1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a **Prometheus** instance, manages its deployment, and provides configuration for it. It also deploys **Alertmanager** and manages **ServiceMonitor** custom resources that specify monitoring definitions for groups of services.
```

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md [hunk 5]
```diff
@@ -21,9 +23,9 @@ The diagram presents monitoring components and the way they interact with one an
...
 
-2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. To obtain such metrics, Prometheus uses the [**kube-state-metrics**](https://github.com/kubernetes/kube-state-metrics) service. It generates the metrics from Kubernetes API objects and exposes them on the `/metrics` HTTP endpoint.  
+2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. To obtain such metrics, Prometheus uses the [**kube-state-metrics**](https://github.com/kubernetes/kube-state-metrics) service. It generates the metrics from Kubernetes API objects and exposes them on the `/metrics` **HTTP endpoint**.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:28]:
```suggestion
2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. Metrics specific to applications (like orders processed per second) are polled from the application pods. Metrics common for all applications are scraped from specialized pods like the **node-exporter**. 
```

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md [hunk 6]
```diff
@@ -21,9 +23,9 @@ The diagram presents monitoring components and the way they interact with one an
...
+2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. To obtain such metrics, Prometheus uses the [**kube-state-metrics**](https://github.com/kubernetes/kube-state-metrics) service. It generates the metrics from Kubernetes API objects and exposes them on the `/metrics` **HTTP endpoint**.  
 Pods can also contain applications with custom metrics, such as the total storage space available in the MinIO server. Prometheus stores this polled data in a time-series database (TSDB) and runs rules over them to generate alerts if it detects any metric anomalies. It also scrapes metrics provided by [**Node Exporter**](https://github.com/mindprince/nvidia_gpu_prometheus_exporter) which exposes existing hardware metrics from external systems as Prometheus metrics.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:29]:
```suggestion
Prometheus stores this polled data in a time-series database (TSDB) and runs rules over them to create aggregated metrics or generate alerts if it detects any metric anomalies. You can use **PrometheusRules** to define alert conditions for metrics. Kyma provides a set of out-of-the-box alerting rules. The definitions of such rules specify the alert logic, the value at which alerts are triggered, the alerts' severity, and more.
```

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md [hunk 7]
```diff
@@ -32,6 +34,6 @@ Pods can also contain applications with custom metrics, such as the total storag
 
 4. [**Alertmanager**](https://prometheus.io/docs/alerting/alertmanager/) receives alerts from Prometheus and forwards this data to configured Slack or Victor Ops channels.  You can use **PrometheusRules** to define alert conditions for metrics. Kyma provides a set of out-of-the-box alerting rules that are passed from Prometheus to Alertmanager. The definitions of such rules specify the alert logic, the value at which alerts are triggered, the alerts' severity, and more.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:35]:
```suggestion
4. [**Alertmanager**](https://prometheus.io/docs/alerting/alertmanager/) receives alerts from Prometheus and forwards this data to configured notification channels like Slack or Victor Ops. 
```

**Review by a-thaler — CHANGES_REQUESTED** [review]

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md [hunk 8]
```diff
@@ -2,8 +2,7 @@
 title: Tracing Architecture
 ---
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:3]: @a-thaler Is this more like it?

image

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md [hunk 9]
```diff
@@ -2,8 +2,7 @@
...
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:4]: Draft:
`
The process of collecting traces by Jaeger looks as follows:

1. The application receives a request, either from an internal or external source.
2. If the application has Istio injection enabled, [Istio proxy](https://github.com/istio/proxy) propagates the correct [HTTP headers](docs/05-technical-reference/other-tracing-envoy-http-headers.md) of the requests to the Jaeger Deployment. Istio proxy calls Jaeger using the [Zipkin](https://zipkin.io/) service, which exposes a Jaeger port compatible with the Zipkin protocol.  
3. Jaeger stores the trace data on a PersistentVolume and makes the trace information available using an API and UI.`

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md (hunk 1 above)
Comment (NHingerl) [reply docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:5]: Suggestion:

> The process of querying traces from Jaeger looks as follows:
> 
> 1. A Kyma user accesses the Jaeger UI to look for specific traces.
> 2. The user has [two options](../../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) to pass the request to the Jaeger Query service and retrieve the trace information: 
>   - If port forwarding has been set up, the user can access the Jaeger UI directly.
>   - If authentication using a OIDC provider has been configured, the request goes through the [Istio Ingress Gateway](TO_DO), which forwards the incoming connections to the service. Jaeger Query passes the request to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
> 3. The Jaeger Deployment provides trace information.

File: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md (hunk 2 above)
Comment (NHingerl) [reply docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md:9]: Thanks for spotting it! Done

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md [hunk 10]
```diff
@@ -16,15 +15,15 @@ Collection and query may occur at the same time. This way, you can inspect speci
 The process of collecting traces by Jaeger looks as follows:
 
 1. The application receives a request, either from an internal or external source.
-2. If the application has Istio injection enabled, [Istio proxy](https://github.com/istio/proxy) propagates the correct [HTTP headers](docs/05-technical-reference/other-tracing-envoy-http-headers.md) of the requests to the Jaeger Deployment. Istio proxy calls Jaeger using the [Zipkin](https://zipkin.io/) service which exposes a Jaeger port compatible with the Zipkin protocol.  
-3. Jaeger processes the data. Specifically, the Jaeger Agent component receives the spans, batches them, and forwards to the Jaeger Collector service.
-4. The BadgerDB database stores the data and persists it using a PersistentVolume resource.
+2. If the application has Istio injection enabled, [Istio proxy](https://github.com/istio/proxy) propagates the correct [HTTP headers](docs/05-technical-reference/other-tracing-envoy-http-headers.md) of the requests to the Jaeger Deployment. Istio proxy calls Jaeger using the [Zipkin](https://zipkin.io/) service, which exposes a Jaeger port compatible with the Zipkin protocol.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:18]:
```suggestion
2. If the application has Istio injection enabled, [Istio proxy](https://github.com/istio/proxy) enriches the request with the correct [HTTP headers](docs/05-technical-reference/other-tracing-envoy-http-headers.md) if missing and propagates them to the Application container. Furthermore Istio proxy sends the trace data for any intercepted request to Jaeger using the [Zipkin](https://zipkin.io/) service, which exposes a Jaeger port compatible with the Zipkin protocol.  
```

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md [hunk 11]
```diff
@@ -16,15 +15,15 @@ Collection and query may occur at the same time. This way, you can inspect speci
...
-3. Jaeger Query passes the request to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
-4. Finally, the functionality provided by the Jaeger Deployment allows you to retrieve trace information.
+2. The user has [two options](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) to pass the request to the Jaeger Query service and retrieve the trace information:
+   - If port forwarding has been set up, the user can access the Jaeger UI directly.
+   - If authentication using a OIDC provider has been configured, the request goes through the [Istio Ingress Gateway](TO_DO), which forwards the incoming connections to the service. Jaeger Query passes the request to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:28]:
```suggestion
   - If authentication using a OIDC provider has been configured, the request goes through the [Istio Ingress Gateway](TO_DO), which forwards the incoming connections to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
```

**Review by a-thaler — CHANGES_REQUESTED** [review]

File: docs/01-overview/02-main-areas/observability/obsv-01-telemetry-observability-overview.md [hunk 12]
```diff
@@ -17,7 +17,7 @@ The collected telemetry data are exposed so that you can view and analyse them w
 Kyma supports a set of tools for in-cluster observability.
 We recommend that you also implement an observability solution of your choice outside your cluster, which has the advantage that you can use the data for troubleshooting and root cause analysis while your cluster is down (also, it doesn't eat into your applications' bandwith).
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply docs/01-overview/02-main-areas/observability/obsv-01-telemetry-observability-overview.md:18]: I thought I'd thrown this bit out already. I'll rephrase the entire paragraphs. However, this is not about the diagrams so I'm not sure this is the place to fix it.

[the remark this answers is not in the document]
Comment (NHingerl) [reply docs/01-overview/02-main-areas/observability/obsv-01-telemetry-observability-overview.md:18]: => new PR https://github.com/kyma-project/kyma/pull/11640

File: docs/05-technical-reference/03-architecture/obsv-02-architecture-logging.md [hunk 13]
```diff
@@ -2,21 +2,15 @@
...
-5. The Loki server processes the log data and stores it in the log store. The labels are stored in the index store.
-6. The user queries the logs using the following tools:
-
-    * Grafana dashboards to analyze and visualize logs fetched and processed by Loki. 
-    * API clients to query log data using the [HTTP API](https://github.com/grafana/loki/blob/master/docs/sources/api/_index.md) for Loki.
-    * Log UI, accessed from the Kyma Console, to display and analyze logs.
+2. The agent detects any new log files in the folder and tails them.
+3. The agent queries the [Kubernetes API Server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) for additional Pod metadata, such as Pod annotations and labels.
+4. The agent enriches log data with labels and sends them to the Loki server. For faster data processing, log data is organized in log chunks. A log chunk consists of metadata, such as labels, collected over a certain time period.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-02-architecture-logging.md:14]:
```suggestion
4. The agent enriches log data with labels and sends them to the Loki server. 
```

Sounds wrong. As it is not important, let's drop it

File: docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md [hunk 14]
```diff
@@ -16,15 +15,15 @@ Collection and query may occur at the same time. This way, you can inspect speci
...
-3. Jaeger Query passes the request to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
-4. Finally, the functionality provided by the Jaeger Deployment allows you to retrieve trace information.
+2. The user has [two options](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) to pass the request to the Jaeger Query service and retrieve the trace information:
+   - If port forwarding has been set up, the user can access the Jaeger UI directly.
+   - If authentication using a OIDC provider has been configured, the request goes through the [Istio Ingress Gateway](TO_DO), which forwards the incoming connections to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
```
Comment (a-thaler) [inline docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md:28]:
```suggestion
   - If authentication using a OIDC provider has been configured, the request goes through the [Istio Ingress Gateway](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/), which forwards the incoming connections to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
```

**Review by a-thaler — APPROVED** [review]

**Comment by NHingerl** [comment]
Remaining changes with https://github.com/kyma-project/kyma/pull/11679

**Pull request merged**